robotoff/spellcheck/percentages/__init__.py

```diff
@@ -0,0 +1,85 @@
+import re
+from typing import List
+
+from robotoff.spellcheck.base_spellchecker import BaseSpellchecker
+
+PERCENTAGE_REGEX = re.compile(
+    r"(\A|.)([0-9]{0,2})([ ]{0,1}?[,|.|;|/]{0,1}[ ]{0,1})([0-9]{0,2})[ ]?(?:%|(?:[\?|/|\\](?:\D|\Z)))"
+)
+ADDITIVES_REGEX = re.compile(r"(?:E ?\d{3,5}[a-z]*)", re.IGNORECASE)
+
+
+class PercentagesSpellchecker(BaseSpellchecker):
+    def correct(self, text: str) -> str:
+        return self.format_percentages(text)
+
+    @staticmethod
+    def format_percentages(text: str) -> str:
```

Comment:
Can you add a docstring to explain what this module does and how it does it?

Reply:
https://github.com/openfoodfacts/robotoff/pull/181/commits/c2840c90ab6849d0182a9bbbb0c951d1465f5156